src/foam/nanos/notification/email/EmailTemplate.js

```diff
@@ -48,7 +48,13 @@ foam.CLASS({
       class: 'String',
       name: 'body',
       documentation: 'Template body',
-      view: { class: 'foam.u2.tag.TextArea', rows: 40, cols: 150 }
+      view: {
+        class: 'foam.u2.MultiView',
+        views: [
+          { class: 'foam.u2.HTMLView' },
+          { class: 'foam.u2.tag.TextArea', rows: 40, cols: 130 }
```

Comment:
```suggestion
          { class: 'foam.u2.tag.TextArea', row: 40 }
```

Jin caught this:
just one thing, I don’t think we want to manually set cols for textarea
it will oveflow in detailview if the screen is too small